README.md

```diff
@@ -27,7 +27,7 @@ If the weekly executive needs to onboard a new collateral:
 $ make
 ```
 
-### Test (DappTools with Optimizations)
+### Test (DappTools without Optimizations)
```

Comment:
Optimizer should default to `on` with the `make test`

https://github.com/makerdao/spells-goerli/blob/PE-1004-20220527/scripts/test-dssspell.sh#L20

Reply:
that looks currently commented/disabled

Reply:
Considering that `make test-dev` is also currently present (where `optimizer` is set to zero), we could consider to keep `make test` optimizer on by default and uncomment the export block.

Reply:
We stopped using the optimizer due the time taking to compile. But honestly I think that delay was with 1 run and wasn't happening with the default 200 runs. No idea why.
We should double check it and if actually works fine with 200 runs renable it here and on mainnet.

Reply:
Maybe we can leave those blocks uncommented and set the default optimizer state to off, then.

IIRC this was a solc issue and not a dapp issue. Not sure if we're still experiencing it with foundry though.

Reply:
Done in `3a83e91`